tsconfig.json

```diff
@@ -0,0 +1,71 @@
+{
+  "ts-node": {
+    "transpileOnly": true,
+    "files": true,
+    "compilerOptions": {
+      "target": "ES2020",
+      "module": "CommonJS",
+      "lib": ["ES2020"]
+    }
+  },
+
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "jsx": "preserve",
+    "baseUrl": ".",
+    "paths": {
+      "@/components/*": [
+        "src/components/*"
+      ],
+      "@/contents/*": [
+        "src/contents/*"
+      ],
+      "@/enums/*": [
+        "src/enums/*"
+      ],
+      "@/styles/*": [
+        "src/styles/*"
+      ],
+      "@/redux/*": [
+        "src/redux/*"
+      ],
+      "@/public/*": [
+        "public/*"
+      ],
+      "@/hooks/*": [
+        "src/hooks/*"
+      ],
+      "@/types/*": [
+        "@types/*"
+      ],
+      "@/services/*": [
+        "src/services/*"
+      ]
+    },
+    "incremental": true
+  },
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "**/*.mdx",
+  ],
+  "exclude": [
+    // "node_modules"
```

Comment:
Is this intentional here ?

Reply:
the whole file is copied from riltech.co tbh

Reply:
Should be excluded probably